comfyui_pano_suite/core/painting.py

```
@@ -0,0 +1,778 @@
+
+
+def _uv_bbox_to_pixels(bbox: dict, tf: dict, width: int, height: int) -> tuple[int, int, int, int, int, int]:
+    """Return (x0, y0, x1, y1, dst_w, dst_h) after applying translation transform."""
+    du = float(tf.get("du") or 0.0)
+    dv = float(tf.get("dv") or 0.0)
+    u0 = ((float(bbox.get("u0", 0.0)) + du) % 1.0 + 1.0) % 1.0
+    v0 = max(0.0, min(1.0, float(bbox.get("v0", 0.0)) + dv))
+    u1 = ((float(bbox.get("u1", 1.0)) + du) % 1.0 + 1.0) % 1.0
+    v1 = max(0.0, min(1.0, float(bbox.get("v1", 1.0)) + dv))
+    wraps = u1 <= u0
+    x0, y0 = int(round(u0 * width)), int(round(v0 * height))
```

Ruff (0.15.5)

[warning] Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)